include/SugarFields/Fields/EmailBody/SugarFieldEmailBody.php

```diff
@@ -64,4 +64,55 @@ public function getDetailViewSmarty($parentFieldArray, $vardef, $displayParams,
 
         return $this->fetch($this->findTemplate('DetailView'));
     }
+
+    /**
+     * @param string $parentFieldArray
+     * @param array $vardef
+     * @param array $displayParams
+     * @param integer $tabindex
+     * @return string
+     */
+    public function getEditViewSmarty($parentFieldArray, $vardef, $displayParams, $tabindex)
+    {
+        $sugarCleaner = new SugarCleaner();
+        $vardef['value'] = $sugarCleaner::cleanHtml($this->getVardefValue($vardef));
+
+        $this->setup($parentFieldArray, $vardef, $displayParams, $tabindex);
+
+        return $this->fetch($this->findTemplate('EditView'));
+    }
+
+    /**
+     * @param string $parentFieldArray
+     * @param array $vardef
+     * @param array $displayParams
+     * @param integer $tabindex
+     * @return string
+     */
+    public function getSearchViewSmarty($parentFieldArray, $vardef, $displayParams, $tabindex)
+    {
+        $sugarCleaner = new SugarCleaner();
+        $vardef['value'] = $sugarCleaner::cleanHtml($this->getVardefValue($vardef));
+
+        $this->setup($parentFieldArray, $vardef, $displayParams, $tabindex);
+
+        return $this->fetch($this->findTemplate('DetailView'));
+    }
+
+    /**
+     * @param array $vardef
+     * @return mixed
+     */
+    private function getVardefValue($vardef)
+    {
+        if (empty($vardef['value'])) {
+            if (!empty($vardef['default'])) {
+                return $vardef['default'];
+            } elseif (!empty($vardef['default_value'])) {
+                return $vardef['default_value'];
+            }
+        }
+
+        return utf8_decode($vardef['value']);
```

Comment:
Please check and handle the case when `$vardef['value']` and/or default/default_value is not set. For e.g what happening if a $vardef has non of these: 
`if (!isset($vardef['value']) && !isset($vardef['default']) && !isset($vardef['default_value'])) ... //??`